### ISSUE TITLE
Exporting collection or getting a collection link exports collection variables using wrong schema

### ISSUE THREAD

**Issue by shamasis** [a85/POSTMan-Chrome-Extension#5237]
**Describe the bug**
The `disabled:true` property is missing on collection variables (and maybe on other variable types too) when collection is exported or link produced.

Refer: https://www.getpostman.com/collections/9be3b455afa362628e70

```
   "variables": [
        {
            "key": "protocol",
            "value": "https",
            "description": "",
            "enabled": true
        },
        {
            "key": "domain",
            "value": "postman-echo.com",
            "description": "",
            "enabled": true
        },
        {
            "key": "port",
            "value": "443",
            "description": "",
            "enabled": true
        },
        {
            "key": "port_start",
            "value": "1",
            "description": "",
            "enabled": false
        },
        {
            "key": "port_end",
            "value": "65536",
            "description": "",
            "enabled": false
        }
    ],
```

**Expected behavior**
Expected `disabled: false` in the last two variables.

**App information (please complete the following information):**
 - App Type: Postman for Mac
 - Postman Version: Version 6.3.1-beta180918-2332
 - OS: OS X 17.7.0 / x64

**Additional context**
Refer: https://schema.getpostman.com/json/collection/v2.1.0/docs/index.html

**Comment by harryi3t** [comment]
This is fixed in our [canary](https://www.getpostman.com/canary) build and will be available in the next production release.

**Comment by harryi3t** [comment]
This is fixed in version 6.4.x
